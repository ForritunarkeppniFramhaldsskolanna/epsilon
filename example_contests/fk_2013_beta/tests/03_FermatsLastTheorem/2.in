16
5
6
7
8
9
42
100
1337
489123
12893121349832
3289142849112581348945
1234419249812479812498
34589324598235798325
23988449812449872394134
3294843759832759815968449864596845498
234253598434895424895649864984356178965479865779845679845267894546897457798546978754896732897
