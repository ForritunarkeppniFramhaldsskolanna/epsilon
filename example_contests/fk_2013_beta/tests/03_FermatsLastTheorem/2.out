no solutions when n = 5
no solutions when n = 6
no solutions when n = 7
no solutions when n = 8
no solutions when n = 9
no solutions when n = 42
no solutions when n = 100
no solutions when n = 1337
no solutions when n = 489123
no solutions when n = 12893121349832
no solutions when n = 3289142849112581348945
no solutions when n = 1234419249812479812498
no solutions when n = 34589324598235798325
no solutions when n = 23988449812449872394134
no solutions when n = 3294843759832759815968449864596845498
no solutions when n = 234253598434895424895649864984356178965479865779845679845267894546897457798546978754896732897
